solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -31,43 +30,22 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
     mapping (IERC20Token => ISmartToken) private reservesToPoolTokens;  // maps each reserve to its pool token
     mapping (ISmartToken => IERC20Token) private poolTokensToReserves;  // maps each pool token to its reserve
 
-    uint8 public amplificationFactor = 20;  // factor to use for conversion calculations (reduces slippage)
-    uint256 public externalRatePropagationTime = 1 hours;  // the time it takes for the external rate to fully take effect
-    uint256 public prevConversionTime;  // previous conversion time in seconds
-
-    // factors used in fee calculations
-    uint32 public lowFeeFactor = 200000;
-    uint32 public highFeeFactor = 800000;
+    Fraction public externalRate;           // conversion rate of 1 primary token in secondary tokens
+    uint256 public externalRateUpdateTime;  // last time the conversion rate was updated (in seconds)
 
     // used by the temp liquidity limit mechanism during the beta
     mapping (IERC20Token => uint256) public maxStakedBalances;
     bool public maxStakedBalanceEnabled = true;
 
-    /**
-      * @dev triggered when the amplification factor is updated
-      *
-      * @param  _prevAmplificationFactor    previous amplification factor
-      * @param  _newAmplificationFactor     new amplification factor
-    */
-    event AmplificationFactorUpdate(uint8 _prevAmplificationFactor, uint8 _newAmplificationFactor);
+    uint32 public oracleDeviationFee = 0; // initial oracle deviation fee, represented in ppm

```

Comment:
it's not really the `initial` deviation fee, that property represents the oracle deviation fee and its default value is 0 (btw, I think the default can be 1%)

Reply:
But we have two oracles, so 2%?

Reply:
no I don't think it's related, 1% is a good start as the entire deviation.

Reply:
Fixed.